### PR TITLE
FOX-23521: Update Options with documentation and examples for skip_diarization option

### PR DESCRIPTION
**Pull request by amikofalvy**
- [ ] Feature is live on production now we can update the documentation accordingly

### CODE REVIEW

File: source/models/options.rst [hunk 1]
```diff
@@ -38,11 +40,13 @@ desired_processing     n/a              *CURRENTLY UNSUPPORTED* list of speech-r
     {
       "media_url": "https://support.rev.com/hc/en-us/article_attachments/200043975/FTC_Sample_1_-_Single.mp3",
       "metadata": "sample metadata",
-      "callback_url": "https://www.example.com/callback"
+      "callback_url": "https://www.example.com/callback",
+      "skip_diarization": "false"
```
Comment (kennygea) [inline source/models/options.rst]: I think we need to include in `/GET transcript` endpoint a sample of when `skip_diarization` is `true`

File: source/endpoints/transcript.rst [hunk 2]
```diff
@@ -56,14 +56,25 @@ Code                   Description
...
+
+                        The json format of the transcript for a job with `skip_diarization` set to true will only show a single speaker.
 ---------------------- ---------------------------------------------------------------
  200                    ``Example Value``
 
                         **Accept: text/plain**
 
-                        `Speaker 1:    00:00    Hello there, this is a sample transcript in plain text`
+                        When the submitted job has `skip_diarization` either set to `false` or not set, the transcript will appear as follows:
+                        `Speaker 0    00:01 What is your name?`
```
Comment (amikofalvy) [inline source/endpoints/transcript.rst]: I wasn't able to run ReadTheDocs locally to verify the integrity of this restructured text format.

File: source/models/options.rst (hunk 1 above)
Comment (amikofalvy) [reply source/models/options.rst]: screen shot 2018-10-24 at 10 51 45 am
screen shot 2018-10-24 at 10 51 35 am

Comment (kennygea) [reply source/models/options.rst]: What about JSON format?

Comment (amikofalvy) [reply source/models/options.rst]: Added to the Transcript Model page

**Pull request merged**